docs/requirements/hardware-proposal.md

```diff
@@ -0,0 +1,110 @@
+# Hardware proposal
+
+Current document covers [requirements](requirements.md) in proposal real hardware
```

Comment:
@3mdeb-karolzmyslowski  After reading this, my impression is that all of the hardware presented here, meets all of the requirements presented before? Is it really true?